chromium_src/chrome/install_static/install_modes.cc

```diff
@@ -3,33 +3,45 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this file,
  * You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-#include "chrome/install_static/buildflags.h"
+#include "chrome/install_static/install_modes.h"
 
+#define GetClientsKeyPath GetClientsKeyPath_Unused
+#define GetClientStateKeyPath GetClientStateKeyPath_Unused
+#define GetClientStateMediumKeyPath GetClientStateMediumKeyPath_Unused
+
+#include "../../../../chrome/install_static/install_modes.cc"
+
+#undef GetClientsKeyPath
+#undef GetClientStateKeyPath
+#undef GetClientStateMediumKeyPath
```

Comment:
So, if we copy `GetUnregisteredKeyPathForProduct` implementation into our override, then we won't need these lines at all and, also, won't need to override the .h file. WDYT?

Reply:
Yes, we can do that if we override the entire file. Sorry I didn't understand this was what you meant before. Changed now.